tests/example-syntax-error/expected_results.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": 1,
+  "message": "Compiling source\n\tballerina_exercism/hello_world:0.1.0",
```

Comment:
This error message seems to not be very useful to the student to figure out what went wrong, Could you output more information?

Reply:
I will see if there is more to the stdout here.